cli/tools/fmt.rs

```diff
@@ -624,6 +625,14 @@ fn is_supported_ext_fmt(path: &Path) -> bool {
   }
 }
 
+fn is_located_in_git(path: &Path) -> bool {
+  let re = Regex::new(r"(^|[\\/])\.git[\\/]").unwrap();
```

Comment:
I think we can just do `path.components().any(|c| matches!(c, Some(Component::Normal(OsStr::new(".git"))))` or something like that instead of using Regex, which will be slower (especially compiling regex each time).

Reply:
Ok, i get it. Thanks for your review.